custom_components/healthchecksio/__init__.py

```
@@ -94,3 +98,44 @@ async def async_unload_entry(
         _LOGGER.info("Successfully removed the HealthChecks.io integration")
     return unload_ok
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate an old config entry."""
+    version = config_entry.version
+
+    if version > 2:
+        # This means the user has downgraded from a future version
+        _LOGGER.error(
+            "HealthChecks.io downgraded and current config not compatible with earlier versions. Integration mut be removed and reinstalled."
```

Pylint (3.3.7)

[convention] Line too long (141/100)

(C0301)